canvas_modules/common-canvas/src/common-properties/components/flexible-table/flexible-table.jsx

```diff
@@ -523,6 +525,7 @@ FlexibleTable.propTypes = {
 	selectedRows: PropTypes.array,
 	rowSelection: PropTypes.string,
 	summaryTable: PropTypes.bool,
+	light: PropTypes.bool,
```

Comment:
Instead of checking `{has(this.props, "light") ? this.props.light : true}` why not set a default prop?